src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_requirements.txt

```diff
@@ -1,3 +1,4 @@
-neuroconv
+neuroconv[openephys,plexon,video]
```

Comment:
Hard to tell when next release will be - can you pin to specific commit for now?

Reply:
> Hard to tell when next release will be - can you pin to specific commit for now?


@CodyCBakerPhD I pinned to latest commit, but also kept the row below, not sure how to specify the extra requirements `[openephys,plexon,video]` otherwise.